src/services/staff/staff-formats.ts

```diff
@@ -3,6 +3,43 @@ export interface AttendanceFormat {
     eventId: string;
 }
 
+export interface Location {
+    description: string;
+    tags: string[];
+    latitude: number;
+    longitude: number;
+}
+
+export interface EventData {
+    eventId: string;
+    name: string;
+    description: string;
+    startTime: number;
+    endTime: number;
+    locations: Location[];
+    isAsync: boolean;
+}
+
+// Function to generate a single event entry
+export function generateEvent(eventId: number): EventData {
+    return {
+        eventId: "event " + eventId,
+        name: "test " + eventId,
+        description: "...",
+        startTime: 1696467600,
```

Comment:
could you set these to be during hackillinois feb 23-26 since mobile only displays those dates